<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>453</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Урок 04</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>30</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_read_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="text">
     <string>Открыть на чтение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Russian" country="Russia"/>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="russian_radio_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Русский</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="english_radio_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="open_edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="save_edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_edit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="quit_edit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_open">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_save">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_new">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_quit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>290</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_settings">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройки горячих клавиш</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
